pypeit/core/coadd.py

```diff
@@ -349,6 +349,7 @@ def poly_ratio_fitfunc_chi2(theta, flux_ref, thismask, arg_dict):
     wave_max = arg_dict['wave_max']
     func = arg_dict['func']
     model = arg_dict['model']
+    use_huber = arg_dict['use_huber']
```

Comment:
I would prefer that you removed the option to not use the huber loss function, unless you can demonstrate that this option is actually needed.

Reply:
removed option to not use Huber loss